### PR TITLE
Story/io 151/modify cell years

### PR DESCRIPTION
**Pull request by EngbergJonas**

### CODE REVIEW

**Review by kgunga — CHANGES_REQUESTED** [review]
Removing a value from cell with backspace and then clicking somewhere else removes the cell

File: src/components/Planning/PlanningGroupsTable/ProjectCell.tsx [hunk 1]
```diff
@@ -1,39 +1,366 @@
...
+    req.estPlanningEnd = addYear(planEnd);
+  }
+
+  if (conEnd) {
+    req.estConstructionStart = addYear(conStart);
+    req.estConstructionEnd = addYear(conEnd);
+  }
+
+  for (let i = finances.length - 1; i >= 0; i--) {
+    if (i !== finances.length - 1) {
```
[the remark this answers is not in the document]
Comment (EngbergJonas) [reply src/components/Planning/PlanningGroupsTable/ProjectCell.tsx]: fixed

File: src/components/Planning/PlanningGroupsTable/ProjectCell.tsx [hunk 2]
```diff
@@ -1,39 +1,366 @@
...
+      req.estConstructionStart = null;
+      req.estConstructionEnd = null;
+    }
+  };
+
+  const updateOverlap = () => {
+    if (isStartOfTimeline) {
+      req.estPlanningStart = null;
+      req.estPlanningEnd = null;
+    } else req.estPlanningEnd = removeYear(planEnd);
```
[the remark this answers is not in the document]
Comment (EngbergJonas) [reply src/components/Planning/PlanningGroupsTable/ProjectCell.tsx]: fixed

File: src/hooks/useIsInViewport.ts [hunk 3]
```diff
@@ -9,9 +9,7 @@ const useIsInViewPort = (ref: RefObject<HTMLElement>) => {
     () =>
       new IntersectionObserver(
         ([entry]) => {
-          console.log('entry', entry);
           const bounding = entry.boundingClientRect;
-
           setDimensions(bounding);
```
Comment (kgunga) [inline src/hooks/useIsInViewport.ts:13]: Make dimensions and isInViewPort states one state

**Review by kgunga — CHANGES_REQUESTED** [review]

**Review by kgunga — APPROVED** [review]

**Pull request merged**